test/user-test.js

```diff
@@ -61,83 +87,118 @@ describe('user', () => {
   });
 
   it('should be able to add a recipe to its list of favorites', () => {
-    const recipe = new Recipe(recipeData[0]);
-    
-    user.chooseFavoriteRecipe(recipe);
+    user.chooseRecipe(greenHam, user.favoriteRecipes);
+    expect(user.favoriteRecipes[0]).to.deep.equal(greenHam);
+  });
 
-    expect(user.favoriteRecipes[0]).to.deep.equal(recipe);
+  it('should be able to add a recipe to its list of recipes to cook', () => {
+    user.chooseRecipe(greenHam, user.recipesToCook);
+    expect(user.recipesToCook[0]).to.deep.equal(greenHam);
   });
 
-  it('should only be able to add recipes to its list of favorites', () => {
+  it('should only be able to add recipes to its recipe lists', () => {
     const recipe = 'Delicious food';
     const otherRecipe = 42;
-    const wrongRecipe = ['food'];
-
-    user.chooseFavoriteRecipe(recipe);
-    user.chooseFavoriteRecipe(otherRecipe);
-    user.chooseFavoriteRecipe(wrongRecipe);
-
+    user.chooseRecipe(recipe, user.favoriteRecipes);
+    user.chooseRecipe(otherRecipe, user.recipesToCook);
     expect(user.favoriteRecipes).to.deep.equal([]);
+    expect(user.recipesToCook).to.deep.equal([]);
   });
 
   it('should let the user search for a recipe by name', () => {
-    const recipe1 = new Recipe(recipeData[0]);
-    const recipe2 = new Recipe(recipeData[1]);
-    const recipe3 = new Recipe(recipeData[2]);
-
-    user.chooseFavoriteRecipe(recipe1);
-    user.chooseFavoriteRecipe(recipe2);
-    user.chooseFavoriteRecipe(recipe3);
+    user.chooseRecipe(aPerfectEgg, user.recipesToCook);
+    user.chooseRecipe(aPerfectEgg, user.favoriteRecipes);
 
-    const searchResults = user.searchFavoriteRecipesByName('Map');
+    const searchResults1 = user.searchRecipesByName('A perfect egg', user.recipesToCook);
+    const searchResults2 = user.searchRecipesByName('A perfect egg', user.favoriteRecipes);
 
-    expect(searchResults[0].name).to.equal(recipe2.name);
+    expect(searchResults1[0]).to.equal(aPerfectEgg);
+    expect(searchResults2[0]).to.equal(aPerfectEgg);
   })
 
   it('should be able to convert an ingredient name to its id', () => {
     const wheatFlourID = user.convertIngredientNameToID('wheat');
     const wheatFlourID2 = user.convertIngredientNameToID('whe');
     const wheatFlourID3 = user.convertIngredientNameToID('wheat flour');
-
+    /// this seems like it does too much
```

Comment:
I agree - was trying to account for more sad paths, but we can definitely cut down on the bulk of this.